packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -363,6 +374,8 @@ class NavigatorState extends State<Navigator> {
 
   bool _debugLocked = false; // used to prevent re-entrant calls to push, pop, and friends
 
+  /// Looks up the route with the given name using [Navigator.onGenerateRoute],
```

Comment:
The sentence concludes with 'then [push] that route', so maybe: Look up the route with ..